org.flatpak.am-okay.yml

```diff
@@ -0,0 +1,73 @@
+id: io.github.baldeuniversel.am-okay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk 
+
+command: am-okay
+
+finish-args:
+    - --share=network
+    - --socket=x11
```

Comment:
it's marked as a console application, so why `x11`?

Reply:
I'm going to remove it now .

Reply:
Updated .